lib/template.html

```diff
@@ -1,5 +1,7 @@
 <!-- Begin Jekyll SEO tag v{{ seo_tag.version }} -->
 
+{% assign seo_page_url = page.url | replace:'/index.html','/' | absolute_url %}
+
 {% if page.url == "/" or page.url == "/about/" %}
```

Comment:
I *think* we've run into this with other plugins. This is an edge case when the use sets the permalink to `/index.html` via YAML front matter. We should check for it.